src/script/components/InputBar/ReplyBar.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {FC} from 'react';
+import Icon from 'Components/Icon';
+import Image from 'Components/Image';
+import RestrictedVideo from 'Components/asset/RestrictedVideo';
+import ParticipantMicOnIcon from 'Components/calling/ParticipantMicOnIcon';
+
+import {renderMessage} from 'Util/messageRenderer';
+import {t} from 'Util/LocalizerUtil';
+import {ContentMessage} from '../../entity/message/ContentMessage';
+
+interface ReplyBarProps {
+  replyMessageEntity: ContentMessage | null;
+  onCancel: () => void;
+}
+
+const ReplyBar: FC<ReplyBarProps> = ({replyMessageEntity, onCancel}) => {
+  const replyAsset = replyMessageEntity?.assets?.()[0];
+
+  return (
+    <div className="input-bar__reply" data-uie-name="input-bar-reply-box">
+      <button type="button" className="button-reset-default" onClick={onCancel} data-uie-name="do-close-reply-box">
+        <Icon.Close fill="white" />
+      </button>
+
+      <div className="input-bar__reply__body">
+        <div className="input-bar__reply__vert-bar"></div>
+
+        <div className="input-bar__reply__text">
+          <div className="input-bar__reply__sender-name">
+            <span data-uie-name="label-name-reply-box">{replyMessageEntity?.headerSenderName()}</span>
+
+            {replyMessageEntity?.was_edited() && <Icon.Edit data-uie-name="message-edited-reply-box" />}
```

Comment:
you need to add aria-label here. since the edit icon is not helpful for screen reader. please talk to astrid for this.
@przemvs